.goreleaser.yml

```diff
@@ -0,0 +1,59 @@
+before:
+  hooks:
+    - go mod tidy
+builds:
+- env:
+  - CGO_ENABLED=0
+  goos:
+    - freebsd
+    - openbsd
+    - solaris
```

Comment:
I looked back at Looker download stats over the past 3 months, I think we can retire Solaris, the BSD's, and 386 as defaults, but maybe that is a larger discussion. They are all < 1% or so (~1.25% for 386), and we do occasionally run in to 32 bit bugs on things like ints, etc.

Reply:
Lets leave this as is for now, we should figure this out more holistically for all of TF oss first.